utils.py

```diff
@@ -13,14 +13,12 @@ def create_directory_if_doesnt_exist(path):
     Path(path).mkdir(parents=True, exist_ok=True)
 
 
-def draw_subspectrogram(spectrogram, sub_len):
```

Comment:
I think users should be able to understand that a spectrogram for a chosen length is drawn when they read the args